backend/README.md

```diff
@@ -22,7 +22,7 @@ $ docker-compose exec neo4j migrate
 
 {% endtab %}
 
-{% tab title="Without Docker" %}
+{% tab title="Local" %}
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-17T15:46:53Z" title="Wednesday, April 17th 2019, 5:46:53 pm +02:00">Apr 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -22,7 +22,7 @@ $ docker-compose exec neo4j migrate
 
 {% endtab %}
 
-{% tab title="Without Docker" %}
+{% tab title="Local" %}
```


```suggestion
{% tab title="Withouth Docker" %}
```
Yes, I was also tempted to call it that way first. Because we call it "local installation" when we mean "without docker", right? People who don't know docker won't understand what it means so I called it simply "without docker"